src/pykoala/data_container.py

```diff
@@ -1,16 +1,34 @@
 """
-This module contains the parent classes that represent the data used
-during the reduction process.
+This module contains the base classes that represent the data used
+during the reduction process of IFS data in `pykoala`.
+
+In a nutshell, `pykoala` interprets all sources of data
+(e.g., images, raw stacked spectra, data cubes) as a :class:`DataContainer` (DC).
+All DC include a series of common properties (e.g., ``intensity`` or ``variance``)
+as well as common methods for I/O, data reduction tracking history or data masking.
```

Comment:
...common methods for I/O, data masking, and logging the data reduction history?